src/platform/tests/test-general.c

```diff
@@ -57,7 +57,7 @@ NMTST_DEFINE ();
 int
 main (int argc, char **argv)
 {
-	nmtst_init_assert_logging (&argc, &argv, "INFO", "DEFAULT");
+	nmtst_init_assert_logging (&argc, &argv, "WARN", "DEFAULT");
```

Comment:
That is because we now instantiate the NMWifiUtils a bit earlier, and it logs an INFO line when created. This seems like a reasonable thing to happen and just bumping the test log levels (as other tests do) seems like a more sensible thing to do than asserting on the log. I just can't figure out a good way to document why I changed this.